### PR TITLE
Feat: remove unused crd in chart

### PR DESCRIPTION
**Pull request by Somefive**
### Description of your changes

1. Remove `WorkloadDefinition`, `ScopeDefinition`, `HealthScope`, `Rollout` CRD in chart.
2. Remove `vela-minimal` chart.

Fixes #

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: pkg/controller/core.oam.dev/v1alpha2/application/application_controller_test.go [hunk 1]
```diff
@@ -848,7 +839,7 @@ var _ = Describe("Test Application Controller", func() {
 		Expect(k8sClient.Delete(ctx, app)).Should(BeNil())
 	})
 
-	It("Test rollout trait all related definition features", func() {
+	PIt("Test rollout trait all related definition features", func() {
```
Comment (FogDong) [inline pkg/controller/core.oam.dev/v1alpha2/application/application_controller_test.go:842]: Do we need those tests in the future?

**Review by wonderflow — COMMENTED** [review]
please write an issue about why these crds can be removed.

**Review by wonderflow — APPROVED** [review]

File: pkg/controller/core.oam.dev/v1alpha2/application/application_controller_test.go (hunk 1 above)
Comment (Somefive) [reply pkg/controller/core.oam.dev/v1alpha2/application/application_controller_test.go:842]: Let's keep it for now. If the legacy rollout API is removed in code completely, we can remove this test.

**Comment by Somefive** [comment]
> please write an issue about why these crds can be removed.

Marked in #5903

**Pull request merged**